.github/workflows/publish-runtime.yml

```diff
@@ -31,24 +31,25 @@ jobs:
 
   build-srtool-runtimes:
     needs: ["setup-scripts"]
-    runs-on: self-hosted
+    runs-on: bare-metal
     strategy:
       matrix:
         chain: ["moonbase", "moonriver", "moonbeam"]
         srtool_image:
           - purestake/srtool
         srtool_image_tag:
-          - 1.66.1
+          - 1.69.0-ssh
```

Comment:
why `-ssh` suffix ?

Reply:
To describe that we are including additional ssh information in the docker image (as 1.69.0 was already published)

Reply:
This is required when using private repos

Reply:
The problem is that this variable is used to get the base image paritytech/srtool, so future upgrade of srtool will not work

Reply:
What do you mean ? 
This is just a tag, pointing to the image we want to use ? 
This image is built manually using the moonbeam-srtool.Dockerfile (we should make it automatic but another time).
I don't see what it will break. If we move to 1.70.0 for exemple, we will have to make the 1.70.0-ssh (or whatever tag name you will want at that time)

Reply:
In the file `moonbeam-srtool.Dockerfile`, the line `FROM paritytech/srtool:${SRTOOL_IMAGE_TAG}` will fail because there is no tag with the -ssh suffix in paritytech/srtool image

Reply:
thank you, I'll check that